chains/v2/types/acala_mandala.json

```diff
@@ -1,16 +1,16 @@
 {
-  "runtime_id": 2025,
+  "runtime_id": 2094,
   "types": {
-      "Balance": "u128",
-      "Index": "u32",
-      "Phase": "frame_system.Phase",
-      "Address": "sp_runtime.multiaddress.MultiAddress",
-      "ExtrinsicSignature": "acala_primitives.signature.AcalaMultiSignature",
-      "ParaId": "polkadot_parachain.primitives.Id",
-      "mandala_runtime.Event": "GenericEvent",
-      "mandala_runtime.Call": "GenericCall",
-      "sp_core.crypto.AccountId32": "GenericAccountId",
-      "pallet_identity.types.Data": "Data"
+    "Balance": "u128",
+    "Index": "u32",
+    "Phase": "frame_system.Phase",
+    "Address": "primitive_types.H160",
```

Comment:
There is an error in the script, it is still sp_runtime.multiaddress.MultiAddress
<img width="1624" alt="Screenshot 2022-10-03 at 18 14 23" src="https://user-images.githubusercontent.com/40560660/193585575-aa0c9fe5-e90c-4258-bb7a-03bf6dbcb2d2.png">

Reply:
done